packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -164,6 +166,14 @@ class PaginatedDataTable extends StatefulWidget {
   /// This value defaults to 56.0 to adhere to the Material Design specifications.
   final double columnSpacing;
 
+  /// Whether the widget should display checkboxes for selectable rows.
+  ///
+  /// If true, a [CheckBox] will be placed at the beginning of each row that is
+  /// selectable.
+  ///
+  /// If false, all rows will not display a [CheckBox].
+  final bool showCheckboxColumn;
```

Comment:
ditto (or better, use doc macros/templates to just insert the text automatically in both places)

Reply:
TIL about doc templates :+1: . Done!